### ISSUE TITLE
Side effect on the HoloViews library in JupyterLab

### ISSUE THREAD

**Issue by dclong** [twosigma/beakerx#7690]
I found that the HoloViews library (for visualization) doesn't work as expected (plots are floating over the notebook rather embedded in the notebook) after installation of BeakerX. I tried to reinstall HoloViews after installation of BeakerX, but still came across the same issue. Below are details of related packages. 

Details:
OS: Ubuntu 16.04
JupyterLab: 0.32   (install by conda)
BeakerX: 1.0.0   (install by conda)
HoloViews: 1.10.7   (install by conda)

Perhaps the BeakerX dev can talk to the HoloViews dev to figure what happened?

**Comment by dclong** [comment]
Below is ticket that I filed in the HoloViews repository. 

https://github.com/ioam/holoviews/issues/2876

**Comment by piorek** [comment]
Hi I've tried to reproduce it, but it looks like its fixed now 
![image](https://user-images.githubusercontent.com/743617/51116320-13f45f80-180b-11e9-8a8c-ae034dd549e8.png)

```
JupyterLab v0.35.4
Known labextensions:
   app dir: /home/lukasz/miniconda3/envs/labx/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v0.38.1  enabled  OK
        @jupyterlab/plotly-extension v0.18.1  enabled  OK
        @pyviz/jupyterlab_pyviz v0.6.4  enabled  OK
        beakerx-jupyterlab v1.3.0  enabled  OK*
        jupyterlab_variableinspector v0.1.0 disabled OK*

   local extensions:
        beakerx-jupyterlab: /home/lukasz/code/codete/beakerx/js/lab
        jupyterlab_variableinspector: /home/lukasz/code/codete/jupyterlab-variableInspector

```
```
bokeh                     1.0.0                      py_0    pyviz
holoviews                 1.11.0             pyh39e3cac_0    pyviz
jupyter                   1.0.0                    py37_7  
jupyter_client            5.2.4                      py_0    conda-forge
jupyter_console           6.0.0                    py37_0  
jupyter_core              4.4.0                      py_0    conda-forge
jupyterlab                0.35.4                   py37_0    conda-forge
jupyterlab_server         0.2.0                      py_0    conda-forge
```
@dclong can you check this and provide example code to reproduce if it still exists?

**Comment by dclong** [comment]
@piorek I'm still facing the same issue. I'm using [JupyterLab in Docker](https://github.com/dclong/docker-jupyterhub-ds). The Python packages in the Docker are install using `pip3` instead of `conda`. The screenshot below shows versions of related libraries. 

<img width="650" alt="image" src="https://user-images.githubusercontent.com/824507/51211109-67c57e00-18c9-11e9-8f3e-eec563892aa4.png">

Below are steps to replicate the issue. 

1. Install Docker. 
2. Pull the image `docker pull dclong/jupyterhub-ds`
3. Start a container of the image using the command below. Notice that this will create a user in the container using your user name as both user name and password.
```
docker run -d \
    --name jupyterhub-ds \
    --log-opt max-size=50m \
    -p 8000:8000 \
    -p 5006:5006 \
    -e DOCKER_USER=`id -un` \
    -e DOCKER_USER_ID=`id -u` \
    -e DOCKER_PASSWORD=`id -un` \
    -e DOCKER_GROUP_ID=`id -g` \
    -e DOCKER_ADMIN_USER=`id -un` \
    -e USER_MEM_LIMIT=4G \
    -v /workdir:/workdir \
    -v /home:/home_host \
    dclong/jupyterhub-ds
```
4. Visit the JupyterLab server. If you launch the container on your localhost, then it is at localhost:8000

5. Log into the JuptyerLab server using your user name (on your machine) as both user name and password. 

6. Open a Python Jupyter notebook. 

7. Run the following visualization code in the Python notebook and you will see the issue. 
```
import numpy as np
import holoviews as hv
hv.extension('bokeh')

# create some example data
python=np.array([2, 3, 7, 5, 26, 221, 44, 233, 254, 265, 266, 267, 120, 111])
pypy=np.array([12, 33, 47, 15, 126, 121, 144, 233, 254, 225, 226, 267, 110, 130])
jython=np.array([22, 43, 10, 25, 26, 101, 114, 203, 194, 215, 201, 227, 139, 160])

dims = dict(kdims='time', vdims='memory')
python = hv.Area(python, label='python', **dims)
pypy   = hv.Area(pypy,   label='pypy',   **dims)
jython = hv.Area(jython, label='jython', **dims)

overlay = (python * pypy * jython).options('Area', fill_alpha=0.5)
overlay.relabel("Area Chart") + hv.Area.stack(overlay).relabel("Stacked Area Chart")
```

**Comment by dclong** [comment]
@piorek BTW, what is the version of your nodejs?

**Comment by piorek** [comment]
@dclong folowed your step by step instructions and still cant see it:
![image](https://user-images.githubusercontent.com/743617/51388469-cd17aa00-1b29-11e9-92dd-169d1d85d253.png)

node version:
![image](https://user-images.githubusercontent.com/743617/51388595-3f888a00-1b2a-11e9-9c08-108b3973368b.png)

**Comment by dclong** [comment]
@piorek That's weird. I tried it in Chrome, Firefox and Opera and encounter the same issue (floating charts over the notebook) every time. Which browser are you using and which version is it?

**Comment by piorek** [comment]
@dclong 
![image](https://user-images.githubusercontent.com/743617/51464259-6c74b100-1d65-11e9-9863-dc35b34f7438.png)

**Comment by dclong** [comment]
@piorek I tried again and it seems to me that the issue is gone.

Note: To be more specifically, I didn't encountered the same issue a bunch of times but I figured out that it was due to cached JS in the saved notebooks. After rerunning the visualization code, saving the notebook and refreshing the JupyterLab server, the issue was gone! I think this is the cause the "weird phenomenon" I mentioned in my last message.
